### PR TITLE
Spark rename

### PR DESCRIPTION
**Pull request by annegentle**
Lots of changes, but code should still work after these changes.

### CODE REVIEW

File: LM-5551 intermediate-aci/dne-dci-intermediate-aci-mission1_health-dashboard/dashboard/app/templates/index.html [hunk 1]
```diff
@@ -91,8 +91,8 @@ <h2 id="{{ app }}_health"></h2>
       </ul>
     </div>
     <br>
-    <p>Great job! Let everyone know you completed the mission by posting to Spark.<br>
-    <a href="/verify_to_spark">CLICK ME</a></p>
+    <p>Great job! Let everyone know you completed the mission by posting to Webex Teams.<br>
+    <a href="/verify_to_webex_teams">CLICK ME</a></p>
```
Comment (annegentle) [inline LM-5551 intermediate-aci/dne-dci-intermediate-aci-mission1_health-dashboard/dashboard/app/templates/index.html:95]: Hey John, I'm not sure if this rename will still work - do you think so?

File: LM-5551 intermediate-aci/dne-dci-intermediate-aci-mission1_health-dashboard/dashboard/app/views.py [hunk 2]
```diff
@@ -96,22 +96,21 @@ def fault_update():
...
         return jsonify(get_faults(request.form['app']))
 
-        
-@app.route('/verify_to_spark', methods=["GET"]) 
-def verify_to_spark(): 
-    message = "I completed the ACI Health Dashboard Mission Completed Successfully!" 
-    post_to_spark(message) 
+
+@app.route('/verify_to_webex_teams', methods=["GET"])
+def verify_to_webex_teams():
```
Comment (annegentle) [inline LM-5551 intermediate-aci/dne-dci-intermediate-aci-mission1_health-dashboard/dashboard/app/views.py:101]: I believe that above index.html should work, based on this additional change, does that sound right?

**Comment by annegentle** [comment]
Hey @johnamcdonough thoughts on this one?

**Comment by movinalot** [comment]
No time to review yet, hopefully, the next day or so, :(

**Comment by annegentle** [comment]
Closing - these all were upgraded in the v2 revisions.

**Pull request closed without merging**